src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -164,6 +166,10 @@ public function setContentDisposition($disposition, $filename = '', $filenameFal
         if ('' === $filenameFallback && (!preg_match('/^[\x20-\x7e]*$/', $filename) || false !== strpos($filename, '%'))) {
             $encoding = mb_detect_encoding($filename, null, true);
```

Comment:
since this is a cleaning code, what about doing something like that instead?
`$encoding = @mb_detect_encoding($filename, null, true) ?: '8bit';`